docs/USAGE-GUIDE.md

```diff
@@ -561,6 +561,19 @@ int s2n_config_add_cert_chain_and_key_to_store(struct s2n_config *config,
 
 **s2n_config_add_cert_chain_and_key_to_store** may be called multiple times to support multiple key types(RSA, ECDSA) and multiple domains. On the server side, the certificate selected will be based on the incoming SNI value and the client's capabilities(supported ciphers). In the case of no certificate matching the client's SNI extension or if no SNI extension was sent by the client, the certificate from the **first** call to **s2n_config_add_cert_chain_and_key_to_store** will be selected.
 
+### s2n\_config\_set\_cert\_chain\_and\_key\_defaults
+
+```c
+int s2n_config_set_cert_chain_and_key_defaults(struct s2n_config *config,
+                                               struct s2n_cert_chain_and_key **cert_key_pairs,
+                                               uint32_t num_cert_key_pairs);
+```
+
+**s2n_config_set_cert_chain_and_key_defaults** explicitly sets certificate chain and private key pairs to be used as defaults for each auth method (key type). A "default" certificate is used when there is not an SNI match with any other configured certificate. Only one certificate can be set as the default per auth method (one RSA default, one ECDSA default, etc.). This is called for a specific **s2n_config** object.
```

Comment:
Can you add a comment about how the "clear" or overwrite behavior works? (e.g. what happens when you call this twice, or after adding certs to the store)

Reply:
Yeah, I can add a comment about this. You're providing a list of certs to the API, so it wouldn't really make sense to attempt to call it multiple times in order to add multiple certs as defaults. The behavior is that the most recent call of this API sets those provided certs as the defaults and no other certificates are acting as defaults anymore (assuming the API call succeeds).